packages/web3-core-types/src/types.ts

```diff
@@ -164,14 +165,13 @@ export interface RequestArguments {
     rpcOptions?: RpcOptions;
     providerOptions?: HttpOptions;
     returnType?: ValidTypesEnum;
-    ethVersion?: number;
+    ethVersion?: 2;
```

Comment:
if `ethVersion` is provided, it **has** to `2` since we're not passing it in `web3-eth`

Reply:
Ah true, does it make more sense for it just to be called `ethVersion2` with type `boolean` instead of `ethVersion`?